spec/Classes.tex

```diff
@@ -1160,8 +1160,8 @@ \subsection{Class Destructor}
 \index{destructor!classes}
 
 A class author may specify additional actions to be performed before a class object is
-reclaimed, by defining a class destructor.  A class destructor is a method that has the
-same name as the class prefixed by a tilde.  A class destructor takes no arguments (aside from
+reclaimed, by defining a class destructor.  A class destructor is a method named
```

Comment:
Want to rename "destructors" to "deinitializers" while you're at it?  Or wait until the more major spec change comes along?  If you want to wait, I would add a note to the spec that the rewrite is coming, just in case we forget between now and after doc freeze